packages/amplication-client/src/Entity/EntityListItem.tsx

```diff
@@ -84,8 +102,50 @@ export const EntityListItem = ({
     setConfirmDelete(false);
   }, [setConfirmDelete]);
 
+  const { trackEvent } = useTracking();
+
+  const [updateResourceSettings, { error: updateError }] = useMutation<TData>(
+    UPDATE_SERVICE_SETTINGS
+  );
+  const { handleSubmit } = useSettingsHook({
+    trackEvent,
+    resourceId,
+    updateResourceSettings,
+  });
+
   const handleConfirmDelete = useCallback(() => {
     setConfirmDelete(false);
+    const { serviceSettings } = resourceSettings;
+    const authEntity = serviceSettings.authEntityName;
+
+    if (authEntity === entity.name) {
+      const updateServiceSettings = {
+        authEntityName: null,
```

Comment:
don't you want to spread all keys of serviceSettings ?

Reply:
Done